ast/cinaps/gen_traverse.ml

```diff
@@ -530,7 +540,10 @@ let print_traverse_ml () =
 
 let print_traverse_mli () =
   Print.newline ();
-  let grammars = Astlib.History.versioned_grammars Astlib.history in
+  let grammars =
+    Astlib.History.versioned_grammars Astlib.history
+    |> List.map ~f:(fun (v, grammar) -> (Astlib.Version.to_string v, grammar))
```

Comment:
Let's not map this over all our alists, if we're going to use a Version.t type, let's use it consistently, and only convert to string in the instances where we need to. I think it'll be clearer that way all around.